src/app/api/checkout/route.ts

```diff
@@ -0,0 +1,79 @@
+// NOTE: NextRequest and NextReponse will only work for middleware, and not for 
+// pages or components, so there is a specific command line for it.
+// eslint-disable-next-line
+import { NextRequest, NextResponse } from "next/server";
+import Stripe from 'stripe';
+
+// NOTE: Instead using `require()`, should use `Stripe()`
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!)
+
+
+// eslint-disable-next-line
```

Comment:
What is eslint complaining about here? Normally if you disable linters, you want to leave a comment explaining to other devs why you've put in an exclusion